packages/build/src/js-transform.ts

```diff
@@ -175,33 +177,41 @@ export function jsTransform(js: string, options: JsTransformOptions): string {
   }
   if (options.transformImportMeta === true ||
       (options.transformImportMeta === undefined &&
-       options.transformModulesToAmd === true)) {
+       options.transformModulesToAmd)) {
     if (!options.filePath) {
       throw new Error('Cannot perform importMeta transform without filePath.');
     }
     if (!options.rootDir) {
       throw new Error('Cannot perform importMeta transform without rootDir.');
     }
-    doBabel = true;
+    doBabelTransform = true;
     let relativeURL = relative(options.rootDir, options.filePath);
     if (isWindows()) {
       // normalize path separators to URL format
       relativeURL = relativeURL.replace(/\\/g, '/');
     }
     plugins.push(rewriteImportMeta(relativeURL));
   }
-  if (options.transformModulesToAmd) {
-    if (options.transformImportMeta === false) {
-      throw new Error(
-          'Cannot use transformModulesToAmd without transformImportMeta.');
-    }
-    doBabel = true;
-    plugins.push(...babelTransformModulesAmd);
+  if (options.transformModulesToAmd && options.transformImportMeta === false) {
+    throw new Error(
+        'Cannot use transformModulesToAmd without transformImportMeta.');
```

Comment:
Can we just turn on `transformImportMeta` when `transformModulesToAmd` is specified? Like, at the initial place where we're handed `options`? Or is this a thing that indicates a bug in polymer-build?

Reply:
Done. We now always turn on the import.meta transform when the AMD transform happens, regardless of the `transformImportMeta` option. Previously it would be auto-set only if you left it undefined, but now you can set it to false explicitly and we will override. Previously we threw because it's non-sensical to ask for AMD but not import.meta (but note that the inverse is fine), but maybe just overriding is more reasonable.

I also tried to make the code around this option more clear. I'm not sure if I succeeded.